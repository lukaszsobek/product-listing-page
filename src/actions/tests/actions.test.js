import {
    filterProducts,
    toggleFilter,
    toggleModal,
    updatePage
} from "../";

describe("Correclty creates action for",() => {

    it("filterProducts", () => {
        expect(filterProducts()).toMatchSnapshot();
    });

    it("toggleFilter", () => {
        expect(toggleFilter(1,2)).toMatchSnapshot();
    }); 
    
    it("toggleModal", () => {
        expect(toggleModal("sample")).toMatchSnapshot();
    }); 
    
    it("updatePage", () => {
        expect(updatePage(1,2).toString()).toMatchSnapshot();
    });     

});