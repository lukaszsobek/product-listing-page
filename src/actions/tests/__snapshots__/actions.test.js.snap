// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correclty creates action for filterProducts 1`] = `
Object {
  "type": "FILTER_PRODUCTS",
}
`;

exports[`Correclty creates action for toggleFilter 1`] = `
Object {
  "filterType": 1,
  "filterValue": 2,
  "type": "TOGGLE_FILTER",
}
`;

exports[`Correclty creates action for toggleModal 1`] = `
Object {
  "modal": "sample",
  "type": "TOGGLE_MODAL",
}
`;

exports[`Correclty creates action for updatePage 1`] = `
"dispatch => {
        dispatch((0, _.toggleFilter)(filterType, filterValue));
        dispatch((0, _.filterProducts)());
    }"
`;
